tools/ci-build/publisher/src/package.rs

```diff
@@ -185,14 +184,19 @@ pub async fn discover_package_manifests(path: PathBuf) -> Result<Vec<PathBuf>> {
             let manifest_path = package_path.join("Cargo.toml");
             if manifest_path.exists() {
                 manifests.push(manifest_path);
-            } else {
-                manifests.extend(discover_package_manifests(package_path).await?.into_iter());
             }
+            manifests.extend(discover_manifests(package_path).await?.into_iter());
         }
     }
     Ok(manifests)
 }
 
+/// Discovers and parses all Cargo.toml files that are packages (as opposed to being exclusively workspaces)
+pub async fn discover_packages(fs: Fs, path: PathBuf) -> Result<Vec<Package>> {
```

Comment:
"crate" would be more accurate than "package", no?

Reply:
Not going to rename this since it's referred to as "package" throughout.